### PR TITLE
license update

### PR DESCRIPTION
**Pull request by andreaslarsen**
copied the text from the app

### CODE REVIEW

File: en-US/about/project/license.md [hunk 1]
```diff
@@ -61,6 +61,42 @@ To read the [LGPL](http://www.gnu.org/licenses/lgpl.html) and [GPL](http://www.g
...
+
+Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
+
+Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
+
+The name of the author may not be used to endorse or promote products derived from this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE AUTHORS ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```
Comment (davelab6) [inline en-US/about/project/license.md]: \`\` -> ''

Comment (andreaslarsen) [inline en-US/about/project/license.md]: ?

Comment (JoesCat) [inline en-US/about/project/license.md]: On May 6, 2015 09:08:30 AM Andreas Larsen wrote:
> 
> ?

You need to pay attention to the double quote marks.
If you use a word processor, these tend to add the enhanced version of the 
quote on the end-part. A lot of compilers configurators and parsers are only 
expecting the simpler double quote mark, so please use the simpler version.

Comment (davelab6) [inline en-US/about/project/license.md]: It should be fine with utf8?

Comment (andreaslarsen) [inline en-US/about/project/license.md]: Ah - I had just copied the text and didn't test it properly. Updated now.

**Comment by andreaslarsen** [comment]
@davelab6 looks ok now, or?

**Comment by davelab6** [comment]
I've merged this but I'll review all the licensing information now, in the repo and the site

**Pull request merged**